src/block-components/icon/index.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import { useAttributeEditHandlers, useBlockAttributes } from '~stackable/hooks'
+import {
+	getAttrNameFunction, __getValue, getShapeSVG,
+} from '~stackable/util'
+import { kebabCase } from 'lodash'
+
+/**
+ * Internal dependencies
+ */
+import { Edit } from './edit'
+import { addAttributes } from './attributes'
+import { Style } from './style'
+import { extractSvg } from './util'
+import FontAwesomeIcon from './font-awesome-icon'
+
+/**
+ * WordPress dependencies
+ */
+import { useBlockEditContext } from '@wordpress/block-editor'
+import {
+	useMemo, Fragment, useState, useRef, useEffect, useCallback,
+} from '@wordpress/element'
+import { IconSearchPopover } from '~stackable/components'
+
+const SVGIcon = props => {
```

Comment:
Is it possible to use the existing SVGIcon component in `src/components/svg-icon`?